.gitignore

```diff
@@ -1 +1,27 @@
-config.yaml
+# Vim/Nvim Session file
+Session.vim
```

Comment:
What do you think to avoid ignoring files that aren't from the project in your `/home/user/.gitignore`? Instead of putting here?

Reply:
I agree. Already changed it.